src/components/Alert1.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+function Alert1(props) {
```

Comment:
```suggestion
function Alert(props) {
```

The filename can be `Alert` as well

Reply:
i was trying Alert but it was showing me error while i was importing it